lib/Serialization/Deserialization.cpp

```diff
@@ -1011,6 +1013,29 @@ ProtocolConformanceDeserializer::readNormalProtocolConformance(
       conformingType, proto, SourceLoc(), dc,
       ProtocolConformanceState::Incomplete,
       isUnchecked, isPreconcurrency);
+
+  if (conformance->isConformanceOfProtocol()) {
+    auto &C = dc->getASTContext();
+
+    // Currently this we should only be skipping only be happening for the
```

Comment:
Slightly garbled comment, did you mean this?

```suggestion
    // Currently this should only be happening for the
```

Reply:
yeap, whoops thank you